src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -2,46 +2,193 @@
 
 import java.util.List;
 
+@SuppressWarnings("unchecked")
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final double GROWTH_FACTOR = 1.5;
+    private static final int MAX_LIST_SIZE = 10;
+    private Node<T>[] nodes;
```

Comment:
LinkedList does not have the same structure as ArrayList
remove this field, and refactor your code
```suggestion
```